src/components/Signin.jsx

```diff
@@ -0,0 +1,276 @@
+                                className="btn btn-outline-success rounded-pill shadow"
+                                width="30%"
+                              >
+                                SignIn
+                              </Button>
+                              <Button
+                                type="submit"
+                                className="btn btn-outline-danger rounded-pill shadow"
+                                width="60%"
+                                marginLeft="5px"
+                              >
+                                Forgot your password?
+                              </Button>
+                            </div>
+                            <div className="form-group text-center">
+                              <label className="h5 contact-label">New to Social-Scheduler ? ,then Register here</label>
+                              <Button
+                                type="submit"
+                                className="btn btn-outline-success rounded-pill shadow"
+                                width="30%"
+                              >
+                                SignUp
+                              </Button>
+                            </div>
+                            {/* social media links for login */}
+                            <div>
+                              <AuthLinkWrapper >
+                                <h2><span>OR</span></h2>
+                              </AuthLinkWrapper>
+                              <DisplayForLinks >
+                                  <li>
+                                    <AuthLink  
+                                      color ="white" bgColor="#3B5998" 
+                                      href="#" hoverColor="white"
+                                      hoverBgColor="#203797" className="btn"
+                                    >
+                                      <i className="fa fa-facebook fa-fw"></i> Signin with Facebook
+                                    </AuthLink>
+                                  </li>
+                                  <li >
+                                    <AuthLink 
+                                      bgColor="#2D2C2C" color="white" 
+                                      hoverBgColor="#4A4A4C" href="#" 
+                                      hoverColor="white"    className=" btn"
+                                    >
+                                      <i className="fa fa-github "></i> Signin with Github
+                                    </AuthLink>
+                                  </li>
+                                  <li >
+                                    <AuthLink 
+                                      color="white" bgColor="#0e76a8" 
+                                      href="#" className=" btn" 
+                                      TextAlign="center" IconSize="32px"
+                                      IconColor="#0e76a8" IconFloat="left" 
```

Comment:
i said you so many times, that removing the float will make the icon appear above the content, and not side of the content. it wont be similar to other auth links. please understand.,once try it otherwise, you are not understanding my point.